dts/arm/atmel/samd5x.dtsi

```diff
@@ -0,0 +1,325 @@
+/*
+ * Copyright (c) 2019 ML!PA Consulting GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/i2c/i2c.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x20000000 0x40000>;
+	};
+
+	backup0: memory@47000000 {
```

Comment:
I assume memory@4700000, flash, and device_id are all part of the SoC.  If such please move these nodes inside the SoC container node.

Reply:
Alright, moved them.